src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
I patterned my initial code after other features in pfSense that rewrite at boot-time, specifically `php.ini`. I'm going to stick with that approach until there is a future, more comprehensive approach to file integrity in pfSense. I see no need to complicate this code, here and now, by introducing checksumming when **assumed-good** (not *known-good* h/t @NOYB) data in config.xml is available.